src/sentry/static/sentry/app/stores/organizationEnvironmentsStore.jsx

```diff
@@ -6,7 +6,8 @@ const DEFAULT_EMPTY_ROUTING_NAME = 'none';
 
 const OrganizationEnvironmentsStore = Reflux.createStore({
   init() {
-    this.items = [];
+    this.environments = null;
```

Comment:
In some stores we keep everything in a single property, e.g. `this._state = {}`, that way you would always `trigger()` with the same thing, and more readily realize what you should expect. The downside of this approach is that you'd have to be careful when mutating `this._state` as we don't want to ever mutate the previous object. Either way works though.

Reply:
ahh i see, I'll keep it like this for now but I get rid of some of the duplication by having the store trigger with `this.trigger(this.get())`